api/http/handler/endpoints/endpoint_create.go

```diff
@@ -416,21 +419,26 @@ func (handler *Handler) createUnsecuredEndpoint(tx dataservices.DataStoreTx, pay
 	endpointType := portainer.DockerEnvironment
 
 	if payload.URL == "" {
-		payload.URL = "unix:///var/run/docker.sock"
+		if payload.ContainerEngine == "podman" {
```

Comment:
Let's use a constant based check here, something like:

```suggestion
		if payload.ContainerEngine == portainer.ContainerEnginePodman {
```

Reply:
Sounds good, updated